app.py

```diff
@@ -242,6 +243,25 @@ def __add_write_node_commands(self, context=None):
                     context=context,
                 )
             )
-            
-            
 
+        # Show the convert actions in the Menu if configured to do so
+        if self.get_setting('show_convert_actions'):
+
+            self.engine.register_command(
+                "Convert from Shotgun Write Nodes",
+                self.convert_to_write_nodes,
+                {
+                    "type": "context_menu",
```

Comment:
is there a way for us to provide a tooltip with more information as part of register command? try adding a `description` key.